src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs

```diff
@@ -102,6 +102,10 @@ public void Format(LogEvent logEvent, TextWriter output)
                 {
                     ScalarValue.Render(logEvent.Timestamp, writer, pt.Format, _formatProvider);
                 }
+                else if (pt.PropertyName == OutputProperties.UtcTimestampPropertyName)
+                {
+                    ScalarValue.Render(logEvent.Timestamp.ToUniversalTime(), writer, pt.Format, _formatProvider);
```

Comment:
We might be better off using `logEvent.Timestamp.UtcDateTime`, here.

Although they're superficially the same thing, if your goal is to output ISO 8601 UTC timestamps, a `DateTime` produces better output:

```csharp
Console.WriteLine($"{DateTimeOffset.Now.ToUniversalTime():O}");
// Prints: 2024-04-21T21:23:15.5713997+00:00

Console.WriteLine($"{DateTimeOffset.Now.UtcDateTime:O}");
// Prints: 2024-04-21T21:23:15.5865127Z
```

Reply:
Oh interesting. 8601 is a 3 digit millisecond AFAIK (and without the 'T')  but happy to change this in the morning when I get back to my pc.

Reply:
Done